src/NamingStrategy/MapNamingStrategy.php

```diff
@@ -52,8 +52,21 @@ public static function createFromHydrationMap(array $hydrationMap) : self
     /**
      * @param array<string, string> $extractionMap
      * @param array<string, string> $hydrationMap
+     * @deprecated Use createFromAsymmetricMap
```

Comment:
The tag `deprecated` is good here, but the description should be more detailed. Like:

> This method has been deprecated and will be removed in the future. Please use createFromAsymmetricMap() instead

Reply:
@froschdesign `deprecated` may be useless, see previous comment